src/Interfaces/Options/Shape/IOptionsShapeImage.ts

```diff
@@ -0,0 +1,6 @@
+export interface IOptionsShapeImage {
+    src: string;
+    width: number;
+    height: number;
+    replace_color: boolean;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Identifier 'replace_color' is not in camel case.](https://app.codacy.com/manual/ar3s/tsparticles/pullRequest?prid=5110920)